target/linux/ath79/image/generic.mk

```diff
@@ -418,6 +439,21 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/avm_fritzx490-wasp
```

Comment:
Am I assuming this builds a sysupgrade.bin? 

(This would be because of 
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/image/Makefile#L85-L86)

Ok, so I get that this image is destined to be a initramfs. 

the `IMAGE_SIZE` looks odd in this context because this is usually set to keep the sysupgrade and factory images sizes in check.

In your case, I think the ath79/generic would still build a sysupgrade.bin because of it being set in the Device/Default
that you are inheriting.

https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/image/Makefile#L85-L86

Do you know what would happen if you remove the "images := " so sysupgrade.bin - that you don't need) is no longer being build? (You could repurpose `KERNEL_SIZE` over `IMAGE_SIZE`... Although, I think you would need to add it to the `KERNEL_INITRAMFS := $$(KERNEL) | check-size $$(KERNEL_SIZE)"